### ISSUE TITLE
Add keyboard shortcuts

### ISSUE THREAD

**Issue by aflyingpumpkin** [diamondburned/gtkcord4#94]
The keyboard shortcuts discord has are a godsend for power users dislike using a mouse. Adding them would be a big help.

**Comment by diamondburned** [comment]
I agree. I want to add customizable keyboard shortcuts by using something like [yaegi](https://github.com/traefik/yaegi), but that takes a bit of time.

For now, the only explicit keyboard shortcut is, like, <kbd>Ctrl</kbd>+<kbd>K</kbd>.
